scripts/tests/test_validate_docstrings.py

```diff
@@ -229,6 +229,24 @@ def good_imports(self):
         """
         pass
 
+    def does_nothing(self, x):
```

Comment:
Can you use a consistent name, and if we're not using the parameter remove it, and keep the test as minimal as is needed please.

Reply:
I've reestructured the tests.